libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -86,31 +86,40 @@ void AP_Camera::init()
         CameraType camera_type = get_type(instance);
 
         // check for servo camera
-        if (camera_type == CameraType::SERVO) {
+        if (false) {
```

Comment:
Txs for this.  This part is a bit awkward, could we make it like AP_Mount perhaps where we simply don't create the backend?

I plan to eventually add a healthy() and a pre-arm check.

Reply:
I have a cleanup for this code which moves it to a switch statement; I was trying to keep this patch simple.

That other branch also saves ~260 bytes, so expect to see it soon!

Reply:
> Txs for this. This part is a bit awkward, could we make it like AP_Mount perhaps where we simply don't create the backend?
> 
> I plan to eventually add a healthy() and a pre-arm check.

I've now created a PR for that further cleanup - https://github.com/ArduPilot/ardupilot/pull/23127

I remembered why I *really* didn't want to mix in general cleanups - it means this PR has a whole lot of "*"s in the "bytes changed" information block in the opening comment for this PR - meaning that those builds are unchanged.  That definitely isn't the case after doing more cleanups.